src/email.js

```diff
@@ -0,0 +1,41 @@
+const SMTPServer = require("smtp-server").SMTPServer;
+const MailParser = require('mailparser').MailParser;
+const addrs = require("email-addresses");
+
+const ROOM_ID = "!IfhHtxETxbbvbBgfol:localhost"; // TODO: Join rooms based on alias and get the roomID
+
+
+module.exports.startSMTP = new SMTPServer({
+    secure: false,
+    logger: false,
+    disabledCommands: ['AUTH', 'STARTTLS'],
+    authOptional: true,
+
+    onData: function (stream, session, callback) {
+        let subject, text;
+        let mailparser = new MailParser();
+        let fromAdd = addrs.parseOneAddress(session.envelope.mailFrom.address);
+        // console.log(session.envelope.mailFrom.address)
+
+        mailparser.on('headers', headers => {
+            subject = headers.get('subject');
+        });
+
+        mailparser.on('data', data => {
+            if (data.type === 'text') {
```

Comment:
I'm curious if it's possible to have multiple `data` events before the `end`, as you sometimes see with large emails and streams where you get chunks. In that case, should we append to text?